src/mono/wasm/debugger/DebuggerTestSuite/MiscTests.cs

```diff
@@ -938,6 +938,7 @@ public async Task GetSourceEmbeddedSource()
         [Fact]
         public async Task InspectLocalsUsingClassFromLibraryUsingDebugTypeFull()
         {
+            await SetJustMyCode(false);
```

Comment:
DebugType FULL is the same thing for us that DON'T have debug symbols, because we can't understand this format of debug symbols.
If JUST MY CODE is enabled we will not load the assemblies if they don't have debug symbols.